From 3def815b7dfe4a0064fb428029a032884731822f Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Thu, 12 Oct 2017 11:13:41 +0800
Subject: [PATCH 2632/5242] MLK-16573-1 arm64: dts: imx8qxp-mek: add pca6416
 IO expander support

commit  175e4534da667c2b0a93a8e897aac9e4ddb470fa from
https://source.codeaurora.org/external/imx/linux-imx.git

NXP pca6416 is compatible with TI tca6416.

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 6986f1d..abe7a94 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -463,6 +463,13 @@
 		assigned-clock-rates = <786432000>, <12288000>, <12288000>;
 	};
 
+	pca6416: gpio@20 {
+		compatible = "ti,tca6416";
+		reg = <0x20>;
+		gpio-controller;
+		#gpio-cells = <2>;
+	};
+
 	cs42888: cs42888@48 {
 		compatible = "cirrus,cs42888";
 		reg = <0x48>;
-- 
1.7.9.5

